### ISSUE TITLE
atan2 fwd derivatives

### ISSUE THREAD

**Issue by proteneer** [google/jax#624]
```
NotImplementedError: Forward-mode differentiation rule for 'atan2' not implemented
```

The derivatives of atan2 is pretty straightforward to implement. Can we add support for it?

**Comment by mattjj** [comment]
Sure, want to give it a shot? You'd want to add code right after [this line](https://github.com/google/jax/blob/312aff35866530eb9b76896bdc321e71ba776d4c/jax/lax/lax.py#L1443), and can probably pattern it off of the lines just above it, which call `defjvp` to set up derivative information for `sin` and `cos`. One difference is that since `atan2` takes two arguments, you'll want to write something more like

```python
ad.defjvp(atan2_p,
          lambda g, x, y: ...,
          lambda g, x, y: ...)
```

**Comment by mattjj** [comment]
Oh oops, just saw #625!

**Comment by mattjj** [comment]
Closed by #625!
